schema/swagger-extensions.json

```diff
@@ -1578,6 +1581,14 @@
     "xmsParameterLocation": {
       "type": "string",
       "enum": ["client", "method"]
+    },
+    "xmsMutability": {
+      "type": "array",
+      "items": {
+        "type": "string",
+        "enum": ["create, read", "update"]
```

Comment:
Shouldn't this be `["create","read","update"]` (end double quotes after create and new quotes for read)